### ISSUE TITLE
Compatibility with Overseerr 1.30.0

### ISSUE THREAD

**Issue by ronilaukkarinen** [RemiRigal/Overseerr-Assistant#26]
Upgraded my Overseerr and noticed the buttons are gone from IMDb. Is the extension 1.30.0 compatible? Console flooding "failed to fetch":

```shell
_app-4227d603af4006bf.js:40          Uncaught (in promise) TypeError: Failed to fetch
    at c (_app-4227d603af4006bf.js:40:878409)
    at Object.error (_app-4227d603af4006bf.js:40:882146)
    at _app-4227d603af4006bf.js:40:857192
c @ _app-4227d603af4006bf.js:40
error @ _app-4227d603af4006bf.js:40
(anonymous) @ _app-4227d603af4006bf.js:40
```

**Comment by jce-zz** [comment]
I face the same issue.

**Comment by RemiRigal** [comment]
The issue came from the placement of the Overseerr buttons on pages from IMDb, not from a compatibility issue with Overseerr itself.
I've updated the extension to fix that in `v1.4.5`, it should be available soon. Please let me know if the buttons are back for IMDb.
